buildprocess/configureWebpack.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/*global require*/
```

Comment:
It doesn't get linted by `gulp lint` but your editor will probably lint it (mine does).  I'm not sure exactly what you're asking though.  Are you asking if the `global require` is needed?